carbonmark-api/src/utils/gqlSdk.ts

```diff
@@ -14,6 +15,7 @@ const ENV_VARS = [
   "CARBON_OFFSETS_GRAPH_API_URL",
   "POOL_PRICES_GRAPH_API_URL",
   "SANITY_GRAPH_API_URL",
+  "DIGITAL_CARBON_GRAPH_API_URL",
```

Comment:
@psparacino These envs have been replaced by constants in app.constants.ts

There will be some rework once you pull from staging